### PR TITLE
Implement URL state

### PR DESCRIPTION
**Pull request by mvl22**
This set of changes implements URL state (layers enabled and map position) as per #147, by addition of a hash. E.g.:

#/rnet-simplified,wards,holyrood/#11.52/55.9659/-2.9368

Further work to implement parameters within each layer will follow.

### CODE REVIEW

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
So far it gives the hash for the layer shown. No distinction between network types (fast/quiet) which is essential I think so please implement before we merge. 

Filters can be optional if a lot of work.

Good to see URLs like this though, great progress from my tests locally:

```
#/rnet,data_zones/#10.65/55.943/-3.1844
```

**Comment by mvl22** [comment]
Filters and network types (basically the same thing, both are sub layer parameters) are future work as I say. I’m not sure yet how long this will take, as things like the sliders and the current HTML structure will need some work.

**Comment by Robinlovelace** [comment]
OK, will check @anguscalder on this, probably worth merging this intermediate step, not sure about the relative importance of sub-layer params.

**Comment by Robinlovelace** [comment]
Comments on this @mem48 ? 

Pro is that it's easy to share map with other layers.

Minor con is increase in code complexity and URL length.

+s outweigh -s and we should merge now is my sense.

Great work Martin.

File: index.html [hunk 1]
```diff
@@ -233,6 +233,19 @@ <h2>Layer controls</h2>
 				<input type="hidden" name="quietness" class="updatelayer slider" data-layer="rnet" />
 				<p>&nbsp;</p>
 			</div>
+			<button class="accordion">Cohesive network</button>
```
Comment (Robinlovelace) [inline index.html:236]: Can you put separate features in separate PRs @mvl22 ?

Seeing as we probably want both of these features (URL state for different layers + this new panel) will merge now in the spirit of 'go fast break stuff' and to see how it looks on the test website.

Let's see how it looks..

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**